applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -20,11 +20,12 @@
 
 LOG_MODULE_REGISTER(slm_fota, CONFIG_SLM_LOG_LEVEL);
 
-/* file_uri: scheme://hostname[:port]path */
-#define FILE_URI_MAX	256
-#define SCHEMA_HTTP	"http"
+/* file_uri: scheme://hostname[:port]path[?parameters] */
+#define FILE_URI_MAX	2048
```

Comment:
In download_client, there is CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE=230. I guess that we need to align to each other.

Reply:
Yes, this PR will be used toghether with https://github.com/nrfconnect/sdk-nrf/pull/10442